### PR TITLE
simpler_recom_s3

### LINKED ISSUE #6: Recommendation of possible classes in the S3 default method
https://github.com/Polkas/miceFast/blob/5131d652466d7a9aa8cfb3b6c6fac7496a40b6bd/R/fill_NA.R#L316

### PR DESCRIPTION
**Pull request by ol-oxy**
closes #6 

check and recommend a class in default S3

### CODE REVIEW

File: R/fill_NA_N.R [hunk 1]
```diff
@@ -190,17 +190,19 @@
 
 fill_NA_N <- function(x, model, posit_y, posit_x, w=NULL, logreg=FALSE, times=10) {
 
-  UseMethod('fill_NA_N')
+  if(inherits(x,'data.frame') || inherits(x,'matrix')){
+
+    UseMethod('fill_NA_N')
+
+  } else {stop("wrong data type - it should be data.frame or matrix")}
```
Comment (Polkas) [inline R/fill_NA_N.R:197]: what with data.table, in all cases

File: R/fill_NA.R [hunk 2]
```diff
@@ -187,16 +187,19 @@
 
 fill_NA <- function(x, model, posit_y, posit_x, w = NULL,logreg=FALSE) {
 
-  UseMethod('fill_NA')
+  if(inherits(x,'data.frame') || inherits(x,'matrix')){
+
+    UseMethod('fill_NA')
+
+  } else {stop("wrong data type - it should be data.frame or matrix")}
```
Comment (Polkas) [inline R/fill_NA.R:194]: data.table here too

**Comment by ol-oxy** [comment]
I do not agree, data.table what then with tibble and all tidyverse options

**Comment by Polkas** [comment]
Ok, i agree

**Pull request merged**